pkg/component/gardenerscheduler/gardener_scheduler.go

```diff
@@ -42,6 +49,22 @@ var TimeoutWaitForManagedResource = 5 * time.Minute
 
 // Values contains configuration values for the gardener-scheduler resources.
 type Values struct {
+	// ClientConnection holds values for the client connection.
+	ClientConnection ClientConnection
```

Comment:
I don't think we need to expose this here - let's simply set QPS=100, Burst=130. We never had to change this in the past years.

Reply:
I guess it makes sense 👍🏻